.travis.yml

```diff
@@ -4,41 +4,38 @@ cache:
   - pip
   - ccache
 
-matrix:
-  include:
-    - os: linux
-      python: "2.7"
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-          packages:
-            - g++-7
-      env:
-        - TH_VERSION="0.4.1"
-        - CC="gcc-7"
-        - CXX="g++-7"
-    - os: linux
-      python: "3.6"
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-          packages:
-            - g++-7
-      env:
-        - TH_VERSION="0.4.1"
-        - CC="gcc-7"
-        - CXX="g++-7"
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - cmake
+      - python-dev
+      - python3-dev
+      - g++-7
+
+env:
+  global:
+    - CC=gcc-7
+    - CXX=g++-7
+    - TH_VERSION=0.4.1
+  matrix:
+    - USE_CONDA=false ESPNET_PYTHON_VERSION=2.7
+    - USE_CONDA=true ESPNET_PYTHON_VERSION=2.7
+    - USE_CONDA=true ESPNET_PYTHON_VERSION=3.7
```

Comment:
Why don't you add `USE_CONDA=false ESPNET_PYTHON_VERSION=3.7` ?

Reply:
In travis, there are no python 3.7 default binary (under 3.6) because its ubuntu 14 is too old. 
https://github.com/travis-ci/travis-ci/issues/9815

Maybe we can use `dist: xenial` to use ubuntu 16 (beta feature)

Reply:
I see.